packages/launcher/electron/auto-updater/index.js

```diff
@@ -5,6 +5,8 @@ const { autoUpdater } = require('electron-updater');
 const log = require('electron-log');
 const isDev = require('electron-is-dev');
 
+const getError = require('../getError');
```

Comment:
```suggestion
const getUpdateError = require('../getUpdateError');
```
Наверно лучше так, а то не очень понятно `getError(error)`